sops.go

```diff
@@ -343,6 +347,16 @@ func (tree Tree) Encrypt(key []byte, cipher Cipher) (string, error) {
 					}
 				}
 			}
+			if !tree.Metadata.MACOnlyEncrypted || tree.Metadata.MACOnlyEncrypted && encrypted {
```

Comment:
Why not simplify this to
```suggestion
			if !tree.Metadata.MACOnlyEncrypted || encrypted {
```
?

Reply:
You are right!